### PR TITLE
chore(deps): update to webpack 5

### PR DESCRIPTION
**Pull request by ylemkimon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,11 +74,11 @@
     "style-loader": "^2.0.0",
     "stylelint": "^13.6.1",
     "stylelint-config-standard": "^20.0.0",
-    "terser-webpack-plugin": "^4.0.0",
-    "webpack": "^4.44.0",
+    "terser-webpack-plugin": "^5.0.3",
+    "webpack": "^5.8.0",
```
Comment (kevinbarabash) [inline package.json:78]: I didn't realize that they were already on 5.8.  For some reason I thought that v5 was a recent thing.

**Review by kevinbarabash — APPROVED** [review]
LGTM.  Thanks for upgrading.  I'm glad to see that webpack's API didn't change too much.

**Comment by kevinbarabash** [comment]
I tested the local dev server after resolving the merge conflict and it's working well including HMR.  The screenshotter appears to be working as well.

**Comment by ylemkimon** [comment]
`webpack-dev-server` is still on the beta, so I was thinking of leaving it as WIP, but if you think it's okay, thank you for the review!

**Comment by kevinbarabash** [comment]
I missed `-beta`.  I think it'll be okay though.  Our set up for webpack-dev-server is pretty straight forward.

**Pull request merged**